third_party/xla_client/pjrt_computation_client.cc

```diff
@@ -61,7 +60,7 @@ PjRtComputationClient::PjRtComputationClient() {
     client_ = xla::GetTpuClient(max_inflight_computations).value();
   } else if (device_type == "TPU_C_API") {
     TF_VLOG(1) << "Initializing PjRt C API client...";
-    client_ = std::move(xla::GetCApiClient().value());
+    client_ = std::move(xla::GetCApiClient("TPU").value());
```

Comment:
Did you also test-confirm that the GPU device code works as expected? ([reference](https://github.com/pytorch/xla/pull/4324/files#diff-3e3a9892f3a9c8e105a561f4c3fa0f66f08c4afa5e9e6ea28d5f848e1a8f1cedR67-R76))

CC @ManfeiBai @yeounoh

Reply:
The reference link you have seems to be pointing to this PR (self-link). Did you want to share some other reference? 

Also, it's for TPU client impl (line 61,  `device_type == "TPU_C_API"`).

Reply:
Thanks, address